src/clr_bundle_add.c

```diff
@@ -57,6 +57,7 @@ static void print_help(const char *name)
 	fprintf(stderr, "   -t, --time              Show verbose time output for swupd operations\n");
 	fprintf(stderr, "   -N, --no-scripts        Do not run the post-update scripts and boot update tool\n");
 	fprintf(stderr, "   -b, --no-boot-update    Do not update the boot files using clr-boot-manager\n");
+	fprintf(stderr, "   -D, --max-parallel-pack-downloads=[n] Set the maximum number of parallel pack downloads in parallel\n");
```

Comment:
This should probably *also* be a compile-time option, as well as have this flag added to the readme. I want to get this merged though so it doesn't have to be done for this PR.

Reply:
Added that to the Readme. We can let this to be a compile-time option in a new PR.